doc/swagger.json

```diff
@@ -0,0 +1,89 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "title": "Platform API",
+        "version": "0.1"
+    },
+    "paths": {
+        "/taoPublishing/api/platforms": {
```

Comment:
it works now, but I don't like that we have difference between URL and real path, I mean that we can use strict path `/taoPublishing/api/Platforms`